### ISSUE TITLE
Use of the the link tag in the html head

### ISSUE THREAD

**Issue by ritikamotwani** [frinyvonnick/node-html-to-image#33]
Hi,
I want to use 
```
<link
  href="https://fonts.googleapis.com/css?family=Poppins:400,700,900"
   rel="stylesheet"
 />
```
in my project. But after adding this tag, if I use the font family in the content, none of the content appears in the final image.
Also unable to add local images in the img src. I have to follow this:

```
const image = fs.readFileSync('./image.jpg');
const base64Image = new Buffer.from(image).toString('base64');
const dataURI = 'data:image/jpeg;base64,' + base64Image
 
nodeHtmlToImage({
  output: './image.png',
  html: '<html><body><img src="{{imageSource}}" /></body></html>',
  content: { imageSource: dataURI }
})
```

Are these bugs/new features to be added to the library?

**Comment by frinyvonnick** [comment]
Hi @ritikamotwani 👋 

Thank you for reporting your issue 🙏 

I tried to reproduce your problem with a simple example in which I use your font and an image on the web:

```js
const nodeHtmlToImage = require('./src/index.js')

nodeHtmlToImage({
  output: './image.png',
  html: `<html>
  <head>
    <link
      href="https://fonts.googleapis.com/css?family=Poppins:400,700,900"
      rel="stylesheet"
    />
    <style>
      body {
        font-family: "Poppins";
        font-weight: 700;
      }

      img {
        max-width: 100%;
      }
    </style>
  </head>
  <body>
    Hello world 🙌!
    <img src="https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/cat_relaxing_on_patio_other/1800x1200_cat_relaxing_on_patio_other.jpg" />
  </body>
</html>`
})
  .then(() => console.log('The image was created successfully!'))
```

Here is the result: 

![image](https://user-images.githubusercontent.com/13099512/83109908-81342800-a0c2-11ea-9587-d4125885afab.png)

As you can see the font is the right one and the image is displayed. Can you try this example on your side and tell me if you have the same result?

**Comment by simonschllng** [comment]
Same problem here. My <link> source goes to my server. I do not even see requests coming in.

**Comment by frinyvonnick** [comment]
@simonschllng can you provide your server code?

**Comment by ritikamotwani** [comment]
Hi @frinyvonnick !
Thanks for your reply. 😄 
I can replicate the problem in this code too.
If I send a get request multiple times, the font renders sometimes and sometimes it shows no content as it cannot detect the font family.
<img width="795" alt="Screenshot 2020-05-29 at 1 10 45 AM" src="https://user-images.githubusercontent.com/22127980/83186174-bf285f00-a149-11ea-8bc3-9fc223466876.png">

**Comment by frinyvonnick** [comment]
@ritikamotwani you can try to set the `waitUntil` property to `networkidle0`. It might fix your issue. Can you confirm it?

**Comment by ritikamotwani** [comment]
@frinyvonnick 
No, it doesn't. One in 6 7 times the text disappears from the image

**Comment by frinyvonnick** [comment]
What is your node version?

**Comment by ritikamotwani** [comment]
OS: macOS Catalina 10.15.2
Node Version: v10.19.0

**Comment by sravya-challa-au3** [comment]
I'm facing the same issue, link tag to google fonts are not working. It is showing blank.
Is there any way to use custom fonts from local files?

**Comment by frinyvonnick** [comment]
I opened a pull request in order to fix this issue. Could one of you (@ritikamotwani @sravya-challa-au3) test it? All instructions are in #40. Thank you in advance 🙏

**Comment by frinyvonnick** [comment]
This has been fixed and published in [3.0.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v3.0.0) 🚀
